products/opensuse/main.pm

```diff
@@ -429,6 +429,7 @@ else {
         || load_qam_install_tests()
         || load_extra_tests()
         || load_virtualization_tests()
+        || load_vagrant_tests()
```

Comment:
You should add to your test suite: 

`EXTRATEST=vagrant` and this line won't be needed (And also VAGRANT=1 can be removed). Please re-read my comment from my previous review regarding the extratests mechanism or give it a look [here](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/c31ca8226be6f8b51c0476ff5dc3cc969c46a71a/lib/main_common.pm#L1724) to se how it works :) 

If you decide to not use the extratests mechanism, please document the variable in variables.md.